deepreg/model/loss/image.py

```diff
@@ -82,3 +84,43 @@ def ssd(y_true: tf.Tensor, y_pred: tf.Tensor) -> tf.Tensor:
     :return: shape = (batch,)
     """
     return tf.reduce_mean(tf.square(y_true - y_pred), axis=[1, 2, 3, 4])
+
+
+def global_mutual_information(y_true: tf.Tensor, y_pred: tf.Tensor) -> tf.Tensor:
```

Comment:
I personally would like more comments on the math and the shapes of the tensors to help people to understand this function.

Reply:
> I personally would like more comments on the math and the shapes of the tensors to help people to understand this function.

ideally yes - but this should not be the reason for refuting the merge, as more detailed technical reference has already been provided.

Reply:
Will add the shape of the tensors and show the detailed pages and equations of the pdf. The corresponding part is easy to follow, so I'll just make some short comments on math if it's necessary.